mmdeploy/backend/sdk/export_info.py

```diff
@@ -182,6 +184,96 @@ def get_inference_info(deploy_cfg: mmcv.Config, model_cfg: mmcv.Config,
     return return_dict
 
 
+def get_transform_static(transforms: List) -> Tuple:
+    """Get the static transform information for Elena use.
+
+    Args:
+        transforms (List): transforms in model_cfg
+
+    Return:
+        tuple(): Composed of the static transform information and the tag.
+    """
+
+    # Current only support basic transform
+    supported_type = [
+        'LoadImageFromFile', 'DefaultFormatBundle', 'Resize', 'CenterCrop',
+        'Normalize', 'ImageToTensor', 'Collect', 'Pad'
+    ]
+    # each transform can only appear once
+    cnt = {}
+    for trans in transforms:
+        tp = trans['type']
+        if tp not in supported_type:
+            return None, None
+        if tp in cnt:
+            return None, None
+        cnt[tp] = 1
+
+    common_dtype = None
+    default_args = {
+        'LoadImageFromFile': {
+            'to_float32': False,
+            'color_type': 'color'
+        },
+        'DefaultFormatBundle': {
+            'img_to_float': True
+        },
+        'Resize': {},
+        'CenterCrop': {},
+        'Normalize': {
+            'to_rgb': True
+        },
+        'ImageToTensor': {},
+        'Collect': {},
+        'Pad': {}
+    }
+
+    transform_static = []
+    for trans in transforms:
+        trans_type = trans['type']
+        trans_args = default_args[trans_type]
+        if trans_type == 'LoadImageFromFile':
```

Comment:
Too much `if-else`. We can probably use `decorator` to re-design this function, making it more extensible